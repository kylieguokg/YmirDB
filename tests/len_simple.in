len a
set a 1 2 3
get a
len a
append a 4 5 6
len a
pop a
len a
pluck a 1
pluck a 1
pluck a 1
pluck a 1
pluck a 1
get a
len a
bye
