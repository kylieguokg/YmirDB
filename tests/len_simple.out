> no such key

> ok

> [1 2 3]

> 3

> ok

> 6

> 1

> 5

> 2

> 3

> 4

> 5

> 6

> []

> 0

> bye
